deploy/charts/cert-manager/templates/cainjector-rbac.yaml

```diff
@@ -74,12 +74,12 @@ rules:
   #   see cmd/cainjector/start.go#L113
   # cert-manager-cainjector-leader-election-core is used by the SecretBased injector controller
   #   see cmd/cainjector/start.go#L137
-  - apiGroups: [""]
-    resources: ["configmaps"]
+  - apiGroups: ["coordination.k8s.io", ""]
+    resources: ["leases", "configmaps"]
     resourceNames: ["cert-manager-cainjector-leader-election", "cert-manager-cainjector-leader-election-core"]
     verbs: ["get", "update", "patch"]
-  - apiGroups: [""]
-    resources: ["configmaps"]
+  - apiGroups: ["coordination.k8s.io", ""]
+    resources: ["leases", "configmaps"]
```

Comment:
Bit of a nit, but ideally these would be separated out as e.g. `configmaps` don't exist in `coordination.k8s.io` and `leases` don't exist in the core API group.

Do we also _need_ both? From what I understand, we only use configmaps for leader election - if that is changing then I think it should be in a separate PR (as we'll also need to consider how we handle changing this without having multiple leaders during upgrade too)

Reply:
Added seperate blocks.

I guess needed for old k8s versions.
https://github.com/kubernetes-sigs/controller-runtime/pull/1144#discussion_r480173688